tests/introspection/ApiSelectorTest.cs

```diff
@@ -840,12 +840,14 @@ protected virtual bool Skip (Type type, string selectorName)
 				if (selectorName == "waitUntilExit")
 					return true;
 				break;
+#if !__MACOS__
```

Comment:
If this is just on iOS, this is not the correct file, it should be in tests/introspection/iOS/iOSApiSelectorTest.cs

Reply:
I believe that it was having issues with TV and Watch simulators as well

Reply:
watchOS and tv will go in the iOS entry since they are generated from it.